docs/farming-&-staking/farming/space-acres/space-acres-install.mdx

```diff
@@ -18,11 +23,68 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
 On Linux: follow [these instructions](https://learn.microsoft.com/en-us/cpp/linux/download-install-and-setup-the-linux-development-workload?view=msvc-170#linux-setup-ubuntu-on-wsl).
 :::
 
-## Step 1: Download the Space Acres Installer
+:::note
+Please note that a wallet with a Substrate address is required to run the software.<br />
+To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](docs/farming-&-staking/wallets/subwallet.md) or [Polkadot.js](docs/farming-&-staking/wallets/polkadot.md).
+:::
+
+<Tabs groupId="Space_Acres_Installation">
+
+<TabItem value="linux" label="Linux">
+<br />
+
+# Step 1: Download the Space Acres Installer
+
+<div className={`${styles.buttons} ${styles.flexContainer}`}>
+    <Link
+        className="button button--secondary"
+        to="https://github.com/subspace/space-acres/releases/download/0.1.33/space-acres_0.1.33-1_amd64.deb"
+        style={{ display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'center', textAlign: 'center', padding: '10px' }}>
+        <span style={{ fontSize: '20px' }}>Linux Installer</span>
+        <span style={{ fontSize: '14px' }}>(.DEB)</span>
+        </Link>
+        <Link
+        className="button button--secondary"
+        to="https://github.com/subspace/space-acres/releases/download/0.1.33/space-acres-0.1.33-x86_64.AppImage"
```

Comment:
Sorry I missed that before - we should adjust all links from _https://github.com/subspace/space-acres_ to _https://github.com/autonomys/space-acres_